mozilla/blurts-server
# Migrate "delete unverified subscribers" cron to TS

# References:

Jira: MNTOR-2142
Figma: N/A



# Description

This was essentially just changing the file extension, which was a good one to get started with :) I'm not actually sure if it's actually still possible to create an unverified account (because we only create an account after FxA sends the user back to us, which it does after verification), but I wasn't confident enough to delete the entire job.

# How to test

Manually change an existing account to unverified in your database, then run `npm run dev:cron:db-delete-unverified-subscribers`. Just like when running `node scripts/delete-unverified-subscribers.js` in `main`, this should delete that user. (If the account is new, you might want to add a low value for `DELETE_UNVERIFIED_SUBSCRIBERS_TIMER` in your `.env.local`.)

# Checklist (Definition of Done)

- [ ] I've added a unit test to test for potential regressions of this bug. - N/A, no behaviour change
